.github/workflows/update-test-ubuntu-git.yml

```diff
@@ -0,0 +1,56 @@
+name: Publishes the test-ubuntu-git Container Image
+
+on:
+  # Use an on demand workflow trigger.  
+  # (Forked copies of actions/checkout won't have permission to update GHCR.io/actions, 
+  #  so avoid trigger events that run automatically.)
+  workflow_dispatch:
+    inputs:
+      publish:
+        description:  'Publish to ghcr.io?'
+        type: boolean
+        required: true
+        default: false
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: actions/test-ubuntu-git
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    # Sets the permissions granted to the `GITHUB_TOKEN` for the actions in this job.
+    permissions:
+      contents: read
+      packages: write
+ 
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      # Use `docker/login-action` to log in to GHCR.io. 
+      # Once published, the packages are scoped to the account defined here.
+      - name: Log in to the ghcr.io container registry
+        uses: docker/login-action@v3.0.0
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
```

Comment:
Is `${{ github.actor }}` always the `actions` org?

Reply:
I think it's whomever triggers the workflow, (I can devise a quick test to confirm), but I believe it'll still work as desired.  All the [examples](https://docs.github.com/en/actions/publishing-packages/publishing-docker-images#publishing-images-to-github-packages) use `${{ github.actor }}`.

The explanation is a bit murky, but some documentation is [here](https://docs.github.com/en/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility#about-setting-visibility-and-access-permissions-for-packages)

Reply:
[`github.repository_owner` is another option](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context), but I wanted to try it this way first and see if it works OK.  

I figure it should help make `auth`-related failures in the workflow more clear.  (e.g. joe triggered the workflow, but joe doesn't have the requisite permissions).

It might also be handy to see who published the image in GHCR.io in the event we need to do some forensics.

Reply:
I see other workflows in the `actions` org that also use `github.actor` when pushing to `ghcr.io` including actions/runner.